cli/tests/sample-configs/invalid/scope_invalid_key.toml

```diff
@@ -0,0 +1,7 @@
+#:schema ../../../src/config-schema.json
+
+# Keys of --scope tables have the same constraints as the top-level keys.
```

Comment:
Let's delete this comment from the invalid file, it doesn't have much to do about why it's invalid.